lib/common/compiler.h

```diff
@@ -151,7 +158,7 @@
 
 /* vectorization
  * older GCC (pre gcc-4.3 picked as the cutoff) uses a different syntax */
-#if !defined(__INTEL_COMPILER) && !defined(__clang__) && defined(__GNUC__)
+#if !defined(__INTEL_COMPILER) && !defined(__clang__) && defined(__GNUC__) && !defined(__LCC__)
```

Comment:
Could you please add a comment explaining the presence of this test ?
This is for future contributors, that may only access the code, not the PR.

Reply:
Added a comment in 3cd085c.